.vscode/cspell.json

```diff
@@ -25,7 +25,8 @@
     "*nlohmann-json*",
     "eng/docs/api/assets/**/*",
     "eng/CredScanSuppression.json",
-    "**/test/ut/recordings/*.json"
+    "**/test/ut/recordings/*.json",
+    "**/CMakeLists.txt"
```

Comment:
Rather than ignoring the entire CMakeLists.txt file (which is useful to have spell check on), add the incorrectly flagged word into the dictionary of "words" below.

Which false positive word does cspell complain about here?

Reply:
Looks like it is just `DISCOVERY_TIMEOUT` that is being added in this PR. Consider adding that to the dictionary.

Reply:
it wasn't that , the cspell just hang for no reason while checking the cmake files , no errors reported.